src/middleware/incontact/src/index.js

```diff
@@ -0,0 +1,186 @@
+/**
```

Comment:
to be consistent please follow the format here, where index file contains routes and starting of the server and extracting main functionality to its own folder

Reply:
genesys example is not following such rule. we started incontact implementation when only genesys example were in place. currently we are not planning to do a full rewrite of a project implementation. please consult with @arneshb

Reply:
we are in process of cleaning up genesys but I totally understand your concern